etc/cyclonedds.xsd

```diff
@@ -17,18 +17,18 @@ CycloneDDS configuration</xs:documentation>
 &lt;p&gt;The General element specifying Domain related settings.&lt;/p&gt;</xs:documentation>
     </xs:annotation>
     <xs:complexType>
-      <xs:choice minOccurs="0" maxOccurs="unbounded">
+      <xs:choice minOccurs="0">
         <xs:element ref="config:Compatibility"/>
-        <xs:element ref="config:DDSSecurity"/>
         <xs:element ref="config:Discovery"/>
         <xs:element ref="config:General"/>
         <xs:element ref="config:Internal"/>
         <xs:element ref="config:Partitioning"/>
         <xs:element ref="config:SSL"/>
+        <xs:element maxOccurs="unbounded" ref="config:Security"/>
```

Comment:
This is weird: why is does this one end up as "unbounded" and the rest doesn't?

Reply:
maxOccurs has a default of 1, which seems about right, even if the parser itself doesn't care about it?

Reply:
My main issue is that it is weird that it is different in this one case. Any "normal" configuration I have seen has each section listed at most once, so both maxOccurs="1" and maxOccurs="unbounded" would work as far as I am concerned.

Reply:
Fixed